tests/opt-irc-3/input.dat

```diff
@@ -0,0 +1,44 @@
+#! Compute the IRC for HOOH torsional rotation at the RHF/STO-3G level of theory, with a tightly converged TS and ZMAT input. This isn't working perfectly, so we need a hack that renders this a poor sample !nosample
+# Print
+# the path to a trajectory file for visualization
+# in Jmol.
+# grep '@IRC' on the output to see a nice printout
+# of the progress of your IRC computation.
+
+molecule h2o2 {
+   H   
+   O 1 OH
+   O 2 OO 1 A 
+   H 3 OH 2 A 1 D 
+ 
+    A         =  104.9380499376
+    D         =   -0.0000000000
+    OH        =    1.0008060649
+    OO        =    1.4057357716
+}
+
+# g_convergence needs to be tight for IRCs:
+set g_convergence gau_verytight
+
+set {
+  basis                      sto-3g
+  opt_type                   irc
+  geom_maxiter               10
+  g_convergence              gau_verytight
+  irc_step_size              0.5
+}
+
+frequencies('scf', dertype=1)
+
+# Lower point group from C2v to C2
+h2o2.reset_point_group('c2')
+
+# An ideal test would check the energy at the minima, but there's
+# an unpredictable last step; see #860. Instead, check after 10 steps.
+# We just need to make sure this makes it past the second point.
+try:
+    energy = optimize('scf')
+except psi4.OptimizationConvergenceError as ex:
+    wfn = ex.wfn
+
+compare_values(-148.752629, wfn.energy(), 5, "Energy of IRC point after 10 iterations")  #TEST
```

Comment:
just FYI, `wfn.energy()` isn't rolled out to all the modules. that is, if you run it on a ccsd wfn, you're likely to get the hf energy. `psi4.get_variable('CURRENT ENERGY')` is still the most thorough at present.

Reply:
I agree here, this is unfortunate and could be a good new-dev project.